modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -35,6 +35,9 @@ Most of the scopes in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.
 * `email`: Read-only access to the user's email.
 * `offline_access`: Access to the user's data when they are not logged in.
 
```

Comment:
Note about the list above:
I typically edit for list items (and steps) to not exceed one sentence or fragment, with additional details, clarifications, or definitions in a separate para below the item. I made an exception here because the list items were so short, but I will likely edit differently when I see other types of list items. Just FYI.
We also have a definition list format that is handy; see the style ref.